services/frontend-v3/src/i18n/en_CA.json

```diff
@@ -341,7 +343,13 @@
   "sign.out": "Logout",
   "home": "Home",
 
-  "error.button": "Back to home",
+  "error.redirect.title": "Unexpected error",
+  "error.redirect.subtitle": "Something went wrong",
+  "error.button": "Back to Home",
+  "error.retry": "Retry",
+  "error.save": "Data not saved due to network error",
+  "error.show": "Show error log",
+  "error.hide": "Hide error log",
```

Comment:
Should add all of these variables to fr_CA also

Reply:
done